wallet/src/main/scala/org/bitcoins/wallet/WalletCallbacks.scala

```diff
@@ -53,6 +67,9 @@ trait WalletCallbacks {
 }
 
 /** Callback for handling a processed transaction */
+trait OnBlockTransactionProcessed
```

Comment:
What is different between `OnBlockTransactionProcessed` and `OnTransactionProcessed`